packages/components/src/components/slider/slider.tsx

```diff
@@ -379,9 +378,9 @@ export class Slider {
                         role="slider"
                         id={this.sliderId + '-from'}
                         aria-valuemin={this.min}
-                        aria-valuenow={this.value}
+                        aria-valuenow={`${this.valueFrom} to ${this.valueTo}`}
```

Comment:
definitely… we'll need this — I remember some of these props to be a bit confusing but I'm not sure, we have to check it out.

Besides that… I think the [`aria-valuenow` should contain only the number](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/slider_role#associated_roles_states_and_properties).

According to [this on MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-valuenow), `aria-valuetext` should be used for these cases @felix-ico:

> When the value to be announced, either the actual value or the value as a percent, may not be clear to users, the [aria-valuetext](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-valuetext) attribute should be used to provide a user-friendly representation of the value. When set, the valuetext string is announced instead of the valuenow numeric value. For example, if a slider represents the days of the week, so the day of the week's aria-valuenow is a number, the aria-valuetext property should be set to a string that makes the slider value understandable, such as "Monday".

Reply:
ah nice, just checked the rating-stars implementation we can do it like that here too. I will revert aria-valuenow as well